recipes/polyfempy/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "polyfempy" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+source:
+  url: https://github.com/polyfem/polyfem-python/archive/{{ version }}.tar.gz
+  sha256: e24bff3fd6c4a54f58be5af3be9f3038f378fed5b250a1da95ab75543d549d5c
+
+build:
+  number: 0
+  skip: True  # [py<31]
+  script:
+    - "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
The repo doesn't seem to have a toml file or setup.py, so I am not sure what this is doing

Reply:
The build is a little complicated.

I have 3 repos:
1) with the pure cpp library https://github.com/polyfem/polyfem/
2) with the binding using 1 and pybind11 which contains the setup https://github.com/polyfem/polyfem-python
3) the fork where i have the recipe